stdlog/stdlog-intern.h

```diff
@@ -28,6 +28,14 @@
 #include <sys/un.h>
 #include "stdlog.h"
 
+#ifndef O_CLOEXEC
```

Comment:
i didn't really see a better place to stuff these fallback defines, but i'm not super familiar with the codebase ...